content/en/platform/corda/4.8/open-source/node-docker-deployments.md

```diff
@@ -0,0 +1,194 @@
+---
+aliases:
+- /head/node-docker-deployments.html
+- /HEAD/node-docker-deployment.html
+- /node-docker-deployments.html
+date: '2020-04-07T12:00:00Z'
+menu:
+  corda-os-4-8:
+    identifier: corda-os-4-8-node-docker-deployments
+    parent: corda-os-4-8-corda-nodes-index
+    weight: 1150
+tags:
+- node
+- docker
+title: Docker deployments
+---
+
+# Docker deployments
+
+This page contains a repository of example manual node configuration operations using Docker. Before executing any of these commands,
+ensure the Corda Docker image has been correctly configured. For more information on this, see the [Official Corda Docker Image](docker-image.md).
+
+The node runs in a container. This is similar to executing the jar directly, except with the need to mount directories into the container that a Corda node would normally have. It is also necessary to set up
```

Comment:
Please review this document and make sure you are using `.jar` (not jar) throughout.